src/configuration/xcWebSocketBridgeConfiguration.ts

```diff
@@ -7,6 +7,7 @@ export const Kinds = {
 
 
 export enum Commands {
+    hb,
```

Comment:
Can you point me where it happens? I think it would be a good idea to decorrelate the enum names from the things we get from the server....